DuckDuckGo/OnboardingSummary-iPad.storyboard

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This seemed like an easier way to constrain that view for iPad than trying to do magic with constraints in a single storyboard.